packages/backend-modules/translate/translations.json

```diff
@@ -2235,6 +2235,18 @@
     {
       "key": "api/publikator/memo/notPublished/author/email",
       "value": "zurückgezogen"
+    },
+    {
+      "key": "api/publikator/generateDerivative/commit/404",
+      "value": "Commit nicht gefunden"
+    },
+    {
+      "key": "api/publikator/generateDerivative/canNot",
+      "value": "Commit darf nicht als Derivat generiert werden"
```

Comment:
This message will be exposed to end users in Publikator, correct? It's a bit cryptic imho. (Same with value below)